packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2171,10 +2277,19 @@ export class ContainerRuntime
 	 * @param modernRuntimeMessage - Does this appear like a current ContainerRuntimeMessage?
```

Comment:
Update these `@param` comments?  Not sure what the best way to document these are, can you do `@param args.message` or something...?

Reply:
The only way I know of getting docs that also provide mouse-hover docs is to extract the type of `args` into an interface and doc each param on the interface. Since this is an internal method, I'd rather not, and instead just do `@param args.message`, etc.

Reply:
Created helper type gave it tsdoc comment